website/sidebars.js

```diff
@@ -125,6 +125,31 @@ module.exports = {
                 label: "I18N",
                 items: ["webiny-apps/i18n/introduction"]
             },
+            {
+                type: "category",
+                label: "Headless CMS",
+                items: [
+                    {
+                        type: "category",
+                        label: "Features",
+                        items: [
+                            "webiny-apps/headless-cms/features/content-modeling",
+                            "webiny-apps/headless-cms/features/content-modeling-groups",
+                            "webiny-apps/headless-cms/features/localization",
+                            "webiny-apps/headless-cms/features/graphql-playground",
+                            "webiny-apps/headless-cms/features/environments",
+                            "webiny-apps/headless-cms/features/aliases",
+                            "webiny-apps/headless-cms/features/access-tokens",
+                            // "webiny-apps/headless-cms/features/roles-permissions",
```

Comment:
see previous comments about articles that need to be added here

Reply:
@SvenAlHamad Related to the Device Frontend Agnostic page, do you want to have this page? I didn't write much of info there, should I add a reference to the Headless CMS + Gatsby.js?

Reply:
I don’t think we need a page for that. People will hopefully see our Guides section and pick it up from there. If we see there are questions about it, then we’ll consider adding a page.